crates/core/tedge/src/cli/certificate/upload.rs

```diff
@@ -91,13 +104,13 @@ impl UploadCertCmd {
         };
 
         let res = client
-            .post(post_url)
+            .post(post_url?)
```

Comment:
I didn't really understand the benefit in in doing this `?` here instead of earlier.

Reply:
Yeah, I changed while playing around its not required anymore. I reverted and fixed it.